src/jbuild

```diff
@@ -0,0 +1,22 @@
+(executable
+ ((name squeezed)
+  (public_name xapi-squeezed)
```

Comment:
I think we should leave the `(public_name squeezed)` as before, because this is the name it is known by, and it is also easier to use it in the spec file.